### PR TITLE
Updated docs, added trivial tests

### PR DESCRIPTION
**Pull request by kevmoo**

### CODE REVIEW

File: webdev/pubspec.yaml [hunk 1]
```diff
@@ -12,5 +12,10 @@ dependencies:
   io: ^0.3.2+1
   stack_trace: ^1.9.2
 
+dev_dependencies:
+  test: ^0.12.0
+  test_descriptor: ^1.0.0
```
Comment (nshahan) [inline webdev/pubspec.yaml]: Where is test_descriptor used?

Comment (kevmoo) [reply webdev/pubspec.yaml]: Oh! Not yet...good catch...

File: tool/travis.sh [hunk 2]
```diff
@@ -29,6 +29,11 @@ while (( "$#" )); do
...
     dartfmt -n --set-exit-if-changed . || EXIT_CODE=$?
     ;;
+  test) echo
+    echo -e '\033[1mTASK: test\033[22m'
+    echo -e 'pub run test'
+    pub run test || EXIT_CODE=$?
+    ;;
   *) echo -e "\033[31mNot expecting TASK '${TASK}'. Error!\033[0m"
     EXIT_CODE=1
     ;;
```
Comment (nshahan) [inline tool/travis.sh]: Is this our recommended approach to travis testing now? What happened to the days of just adding a few lines in the travis.yaml?

Comment (kevmoo) [reply tool/travis.sh]: This is using `mono_repo` – let's us test multiple packages in one repo easily...

**Comment by kevmoo** [comment]
PTAL

File: webdev/test/integration_test.dart [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+    var output = (await process.stdoutStream().join('\n')).trim();
+
+    expect(output, contains('Could not find a command named "monkey".'));
+
+    await process.shouldExit(64);
+  });
+
+  test('should fail in a package without a build_runner dependency', () async {
+    var process = await TestProcess.start('dart', ['bin/webdev.dart', 'serve']);
+    await process.shouldExit(255);
```
Comment (nshahan) [inline webdev/test/integration_test.dart]: Are we expecting anything other than a failure yet?

Comment (kevmoo) [reply webdev/test/integration_test.dart]: This is running against itself – so no build_runner dep or anything. We expect it to always fail.

...although in the future it should fail with a more helpful message and a different error code

File: webdev/test/integration_test.dart [hunk 4]
```diff
@@ -0,0 +1,39 @@
+// Copyright (c) 2017, the Dart project authors.  Please see the AUTHORS file
```
Comment (natebosch) [inline webdev/test/integration_test.dart]: 2018

**Comment by kevmoo** [comment]
Eh...

**Pull request merged**